src/simple_cache_client.rs

```diff
@@ -543,10 +561,10 @@ impl SimpleCacheClient {
     ///     momento.delete_cache(&cache_name).await;
     /// # })
     /// ```
-    pub async fn dictionary_set<K: MomentoRequest, V: MomentoRequest>(
+    pub async fn dictionary_set<K: MomentoRequest, V: MomentoRequest, D: MomentoRequest>(
         &mut self,
         cache_name: &str,
-        dictionary_name: &str,
+        dictionary_name: D,
```

Comment:
nit: I don't know if I'm the only one, but these 1char generics on a parameter get me a little. `dictionary_name: impl MomentoRequest,` would read more nicely imo and would make the method signature more friendly.

Reply:
I've updated this for the new parameters added in this PR. If you're open to it, I'll make a second PR to replace existing uses where appropriate.

Reply:
> If you're open to it, I'll make a second PR to replace existing uses where appropriate.

Of course, and with my thanks! ❤️